### PR TITLE
Fix neutron handling for ml2

### PR DESCRIPTION
**Pull request by toabctl**
Due to recent changes in barclamp-neutron, fix core plugin
and ml2 type drivers handling.

### CODE REVIEW

**Comment by vuntz** [comment]
+1

File: chef/cookbooks/hyperv/recipes/config.rb [hunk 1]
```diff
@@ -37,13 +37,14 @@
   neutron_service_user = neutron_server[:neutron][:service_user]
   neutron_service_password = neutron_server[:neutron][:service_password]
   neutron_networking_mode = neutron_server[:neutron][:networking_mode]
-  neutron_networking_plugin = neutron_server[:neutron][:networking_plugin]
```
Comment (rhafer) [inline chef/cookbooks/hyperv/recipes/config.rb]: This looks wrong. ```networking_mode``` got removed from the neutron barclamp.  And ```networking_plugin``` is still there (can be ```vmware``` or ```ml2``` currently).

Comment (toabctl) [inline chef/cookbooks/hyperv/recipes/config.rb]: > This looks wrong. networking_mode got removed from the neutron barclamp. And networking_plugin is still there (can be vmware or ml2 currently).

Yeah. I recognized it, too. Will change that.

**Comment by rhafer** [comment]
+1

**Pull request merged**